barretenberg/cpp/src/barretenberg/dsl/acir_format/recursion_constraint.hpp

```diff
@@ -6,6 +6,9 @@
 
 namespace acir_format {
 
+// Used to specify the type of recursive verifier via the proof_type specified by verify_proof() from noir
```

Comment:
```suggestion
// Used to specify the type of recursive verifier via the proof_type specified by the RecursiveAggregation opcode from ACIR
```
Very tiny nit again: In general it would be good to try and maintain the boundaries of separation in even our comments for clarity sake. Barretenberg should only know about "ACIR". In theory, we could have a different language frontend compile down to ACIR. And the name of the method which lays down the `RecursiveAggregation` opcode can change. e.g even in this PR it was renamed to verify_proof_internal. verify_proof_internal is very close to verify_proof, but we have previously considered just renaming it to `aggregate` and placing it under a recursion module (e.g. `std::recursion::aggregate(...)`.